attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4395 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    	src="https://forms.gle/zbJoTSz3nfYq1VrY6",
</pre><p>Link for Form1: https://forms.gle/eescFtZ9hyqzQ2pD6</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>